content/learn/book/your-first-game/_index.md

```diff
@@ -0,0 +1,685 @@
++++
+title = "Your first game"
+weight = 3
+sort_by = "weight"
+template = "book-section.html"
+page_template = "book-section.html"
++++
+
+Okay, now that you know the basics of Bevy, let's look at an actual game. Specifically let's look at one of the examples that come with Bevy: Breakout.
+
+![screenshot1](breakout1.png)
+
+For those of you who don't know, Breakout is an arcade game developed and published by Atari in 1976. In the game a layer of bricks lines the top third of the screen and the goal is to destroy them all. A ball moves straight around the screen, bouncing off the top and two sides of the screen. When a brick is hit, the ball bounces back and the brick is destroyed. The player loses a turn when the ball touches the bottom of the screen; to prevent this from happening, the player has a horizontally movable paddle to bounce the ball upward, keeping it in play. [Source: Wikipedia](https://en.wikipedia.org/wiki/Breakout_(video_game))
+
+Here is a version of the game compiled to WebGL that you can play right away:  
+[https://mrk.sed.pl/bevy-showcase/breakout.html](https://mrk.sed.pl/bevy-showcase/breakout.html)
```

Comment:
This is *super* useful to have. Not sure about hosting a direct link to an external non-Bevy site for this, as wonderful as it is for you to offer that.

Reply:
I fully agree with your concern. Just for your information, I got that link from the Bevy news site here:
https://bevyengine.org/news/bevy-0-4/#wasm-webgl2
I am not the one having created the example nor am I hosting it.  
Maybe it would be better to link to the news site?  
But I also agree that the value for a beginner is super high, so maybe we can recreate the demo site on an official server location? After all it seems like a "simple" cross compile using the official example code. But I am not sure.